### PR TITLE
give complete path to signature creation

### PR DESCRIPTION
**Pull request by bpike7**
`const queryParams = {
  source_currency: 'USD',
  destination_currency: 'MXN',
}

console.log(`/rates?${new URLSearchParams(queryParams)}`);

// /rates?source_currency=USD&destination_currency=MXN
`

### CODE REVIEW

File: lib/actions.js [hunk 1]
```diff
@@ -201,6 +201,14 @@ function reqInstance(method, path, data) {
     })
 }
 
+function resolveUrlWithParams(url, queryParams) {
+  // take in queryParams as object, and transform into ?key=value&key2=value2
```
Comment (ben-routefusion) [inline lib/actions.js:205]: You should be able to just use the built-in URLSearchParams helper class:
```js
const qs = new URLSearchParams(queryParams);
return [url, qs.toString()].join("?");
```

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams

Comment (bpike7) [reply lib/actions.js:205]: Nice. Ok I'll update

Comment (ben-routefusion) [reply lib/actions.js:205]: `querystring` is another alternative:
```js
const qs = querystring.stringify(queryParams);
```

Comment (bpike7) [reply lib/actions.js:205]: That one was for myself haha. Always looking for an excuse to build a clean reduce implementation lol

**Review by ben-routefusion — APPROVED** [review]

**Pull request merged**